packages/cli-lib/lib/uploadFolder.js

```diff
@@ -132,12 +142,27 @@ async function uploadFolder(accountId, src, dest, options) {
               payload: file,
             })
           );
+          return {
+            resultType: FileUploadResultType.FAILURE,
+            error,
+            file,
+          };
         }
       };
     })
   );
+  return results;
+}
+
+function hasUploadErrors(results) {
+  return (
+    results.filter(result => result.resultType === FileUploadResultType.FAILURE)
```

Comment:
You could probably just use `.some` here so you don't have to check for length

Reply:
Switched over in af6fb96